### PR TITLE
Initial plugin skeleton

### PR DESCRIPTION
**Pull request by advayDev1**
We can move functionality from j2objc.gradle into J2objcPlugin.groovy as we are able to.

```
Skeleton for empty Gradle plugin build
- Empty Groovy class for plugin.
- Standard Gradle build wrapper

For now we use com.github.brunobowden.j2objcgradle as our root package (i.e. group).
```

### CODE REVIEW

**Comment by advayDev1** [comment]
Starts issue #55

File: src/main/resources/META-INF/gradle-plugins/j2objc.properties [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright (c) 2015 the authors of j2objc-gradle (see AUTHORS file)
+#
+
+implementation-class=com.github.brunobowden.j2objcgradle.J2objcPlugin
```
Comment (brunobowden) [inline src/main/resources/META-INF/gradle-plugins/j2objc.properties]: Add explanation on the magic

**Comment by advayDev1** [comment]
ptal

**Comment by confile** [comment]
LGTM

**Comment by brunobowden** [comment]
I gave an LGTM in person

**Pull request merged**